src/vaadin-message.js

```diff
@@ -44,20 +45,61 @@ class MessageElement extends ElementMixin(ThemableMixin(PolymerElement)) {
         type: String,
         reflectToAttribute: true
       },
+
       /**
-       * A user object that can be used to render avatar and name.
-       * The user object can consist of the following properties:
-       * ```js
-       * user: {
-       *   name: string,
-       *   abbr: string,
-       *   img: string,
-       *   colorIndex: number
+       * The name of the user posting the message.
+       * It will be placed in the name part to indicate who has sent the message.
+       * It is also used as a tooltip for the avatar.
+       * Example: `message.userName = "Jessica Jacobs";`
+       */
+      userName: {
+        type: String,
+        reflectToAttribute: true
+      },
+
+      /**
+       * The abbriviation of the user.
+       * The abbreviation will be passed on to avatar of the message.
+       * If the user does not have an avatar picture set with `userImg`, `userAbbr` will be shown in the avatar.
+       * Example: `message.userAbbr = "JJ`
+       */
+      userAbbr: {
+        type: String,
+        reflectToAttribute: true
+      },
+
+      /**
+       * An URL for a user image.
+       * The image will be used in the avatar component to show who has sent the message.
+       * Example: `message.userImg = "/static/img/avatar.jpg";`
+       */
+      userImg: {
+        type: String,
+        reflectToAttribute: true
```

Comment:
There is no need for this property to reflect to attribute. 

Same applies to other properties above (`time`, `userName`, `userAbbr`, `userColorIndex`).
This option generally is only needed for Boolean properties that are used for styling.

Reply:
Done. I modeled it after `vaadin-avatar`, as it is a new component with similar API. [It reflected the attributes](https://github.com/vaadin/vaadin-avatar/blob/master/src/vaadin-avatar.js#L160).

Reply:
Thanks. Not sure why we added those when creating `vaadin-avatar`, somehow I did not notice it there.